plugins/module_utils/rds.py

```diff
@@ -130,15 +137,6 @@ def handle_errors(module, exception, method_name, parameters):
             changed = False
         else:
             module.fail_json_aws(exception, msg='Unable to {0}'.format(get_rds_method_attribute(method_name, module).operation_description))
-    elif method_name == 'create_db_instance' and error_code == 'InvalidParameterValue':
```

Comment:
removed and moved engine validation to community.aws.rds_instance (added choices for valid engines so this code will never be reached)